### DISCUSSION TITLE
algoritmos para la tarea

### DISCUSSION THREAD

**Discussion by Agustinuc** [IIC2133-PUC/2024-1#176] [Tarea 3] [answered]
holaa, me surgio una duda al leer el enunciado. Se debe de hacer con algun tipo de estructura de dato especifico cada parte de la tarea. Gracias

**Comment by 4n1t4** [comment, marked as answer]
Hola, efectivamente la está enfocada a cierta área de algoritmos y estrategias más que a estructuras. Sin embargo,  el uso de estructuras siempre va a depender de tu solución y la forma en que creas lograr una mejor respuesta. :)
